templates/taxi/car_detail.html

```diff
@@ -16,7 +16,19 @@ <h1>Drivers</h1>
   <hr>
   <ul>
     {% for driver in car.drivers.all %}
-      <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
+      <hr>
+      <p> <strong>Username:</strong> {{ driver.username }}</p>
+      <p> <strong>Full name:</strong> {{ driver.first_name }} {{ driver.last_name }}</p>
+      <p> <strong>License number:</strong> {{ driver.license_number }}</p>
+      <hr>
     {% endfor %}
   </ul>
+  <form action="{% url 'taxi:car-detail' car.pk %}" method="post">
+    {% csrf_token %}
+    {% if request.user in car.drivers.all %}
```

Comment:
Ensure that the logic `{% if request.user in car.drivers.all %}` is efficient and does not lead to performance issues. Consider using a more efficient method to check if the user is in the drivers list, especially if the list is large.